### PR TITLE
[SE-3405] Add GDPR endpoint to nginx

### PR DESCRIPTION
**Pull request by pkulkark**
This PR adds an nginx entry for the gdpr user retirement api introduced in [open-craft/edx-platform#300](https://github.com/open-craft/edx-platform/pull/300). Without this entry, nginx will reject the api requests made.

**Dependencies:**
-  [open-craft/edx-platform#300](https://github.com/open-craft/edx-platform/pull/300)

**Sandbox URL:** TBD

**Testing Instructions:**
See testing instructions in [open-craft/edx-platform#300](https://github.com/open-craft/edx-platform/pull/300)

**Reviewers:**
- [ ] @mavidser

### CODE REVIEW

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2 [hunk 1]
```diff
@@ -290,6 +290,10 @@ error_page {{ k }} {{ v }};
     try_files $uri @proxy_to_lms_app;
   }
 
+  location /v1/accounts/gdpr_retire_users {
```
Comment (mavidser) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2:294]: Can you add a comment on top of this explaining that the endpoint uses OAuth for authentication instead?

**Review by mavidser — CHANGES_REQUESTED** [review]

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2 (hunk 1 above)
Comment (nizarmah) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2:294]: Addressed 👍🏼

**Review by mavidser — APPROVED** [review]
LGTM :+1:

- [x] I tested this: Tested that this patch was applied on the sandbox, and the URLs worked as expected
- [x] I read through the code
- [x] Includes documentation

**Comment by nizarmah** [comment]
This has been merged upstream, so merging here.

**Review by nizarmah — APPROVED** [review]
The changes have been merged upstream https://github.com/edx/configuration/pull/6202.

Merging here too 👍🏼

**Pull request merged**